packages/link/src/Link.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+
+type Props = {
+  /** Content of the link. Can be anything */
+  children: MnrlReactNode,
+  /** Link renders an <a> by default but you can specify a different tag or component as the root (e.g. a React Router Link).*/
+  root?: MnrlReactNode
+};
+
+/**
+ * Link component
+ */
+export default function Link({ children, root = 'a', ...otherProps }: Props) {
+  if (otherProps.onClick) {
+    console.error(
+      'You put an onClick on a Link. Maybe you want a Button instead?'
+    );
+  }
+  const Root = createStyledComponent(
+    root,
+    (props, theme) => {
+      return {
+        color: theme.color_link,
+        fontWeight: theme.Link_fontWeight || theme.fontWeight_regular,
+        fontSize: theme.Link_fontSize || `${2 * theme.fontSize_base}px`,
+
+        '&:hover': {
+          color: theme.color_link_hover
+        },
+        '&:focus': {
```

Comment:
This focus style is not even remotely discernible. Please follow the `boxShadow` design used in Button (but with no padding).